components/Layout.js

```diff
@@ -59,6 +59,17 @@ injectGlobal`
     ${subsectionTitleText}
   }
 
+  h4 {
+    ${bodyText}
+    font-size: 1rem;
```

Comment:
Any reason why you don't use the `rem(pxs)` approach here? Would be more consistent with the rest of the code base, and would make it more clear when talking to and tweaking with designers.